stable/admin/templates/_helpers.tpl

```diff
@@ -144,3 +144,40 @@ Import ENV vars from configMaps
 {{- define "admin.envFrom" }}
 envFrom: [{{- range $index, $map := .Values.admin.envFrom.configMapRef }}{{if gt $index 0}},{{end}} configMapRef: { name: {{$map}} } {{ end }}]
 {{- end -}}
+
+{{/*
+Define the cluster domains
+*/}}
+{{- define "cluster.domain" -}}
+{{- .Values.cloud.cluster.domain | default "cluster.local" }}
+{{- end -}}
+
+{{- define "cluster.entrypointDomain" -}}
+{{- .Values.cloud.cluster.entrypointDomain | default (include "cluster.domain" .) }}
+{{- end -}}
+
+{{/*
+Define the fully qualified NuoDB Admin address for the domain entrypoint.
+*/}}
+{{- define "admin.entrypointFullname" -}}
+{{- $domain := default "domain" .Values.admin.domain -}}
+{{- $cluster := default "cluster0" .Values.cloud.cluster.entrypointName -}}
+{{- if .Values.admin.fullnameOverride -}}
+{{- .Values.admin.fullnameOverride | trunc 50 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.admin.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- printf "%s-%s-%s" .Release.Name .Values.admin.domain $cluster | trunc 50 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s-%s-%s" .Release.Name .Values.admin.domain $cluster $name | trunc 50 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "nuodb.domainEntrypoint" -}}
+{{ include "admin.entrypointFullname" . }}-0.{{ .Values.admin.domain }}.$(NAMESPACE).svc.{{ include "cluster.entrypointDomain" . }}
```

Comment:
So the `cluster<ordinal>` tag will be present in every FQDN? Was this already the case using the old variable `clusterName`?

Reply:
I guess this is already the case, but without the ordinal:

```
$ kubectl exec -it edgy-peahen-nuodb-cluster0-admin-0 -- hostname -f
edgy-peahen-nuodb-cluster0-admin-0.nuodb.default.svc.cluster.local
```

Reply:
cluster domain is configured in the k8s install so will always be there, defaulting to cluster.local but requiring changing for multicluster so that DNS can forward correctly